spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/system/OutputCaptureRuleTests.java

```diff
@@ -37,10 +37,27 @@ public void toStringShouldReturnAllCapturedOutput() {
 		assertThat(this.output.toString()).contains("Hello World");
 	}
 
+	@Test
+	public void getAllShouldReturnAllCapturedOutput() {
+		System.out.println("Hello World");
+		assertThat(this.output.getAll()).contains("Hello World");
+	}
+
+	@Test
+	public void getOutShouldReturnAllCapturedOutput() {
+		System.out.println("Hello World");
+		assertThat(this.output.getOut()).contains("Hello World");
+	}
+
 	@Test
 	public void captureShouldBeAssertable() {
 		System.out.println("Hello World");
 		assertThat(this.output).contains("Hello World");
 	}
 
+	@Test
+	public void getErrShouldReturnAllCapturedOutput() {
```

Comment:
I think this should named something like `getErrShouldOnlyReturnOutputCapturedFromSystemErr`